Sources_v3/Controllers/ChannelController_Tests.swift

```diff
@@ -216,6 +216,52 @@ class ChannelController_Tests: StressTestCase {
     }
     
     // MARK: - Delegate tests
+
+    func test_channelMemberEvents_areForwaredToDelegate() throws {
+        let delegate = TestDelegate()
+        controller.delegate = delegate
+
+        // Set the queue for delegate calls
+        delegate.expectedQueueId = controllerCallbackQueueID
+
+        // Simulate `startUpdating()` call
+        controller.startUpdating()
+
+        // Send notification with event happened in the observed channel
+        let event = TestMemberEvent(cid: controller.channelQuery.cid, userId: .unique)
+        let notification = Notification(newEventReceived: event, sender: self)
+        client.webSocketClient.notificationCenter.post(notification)
+
+        // Assert the event is received
+        AssertAsync {
+            Assert.willBeTrue(delegate.didReceiveMemberEvent_event is TestMemberEvent)
+            Assert.willBeEqual(delegate.didReceiveMemberEvent_event?.userId, event.userId)
+            Assert.willBeEqual(delegate.didReceiveMemberEvent_event?.cid, event.cid)
```

Comment:
You can make `TestMemberEvent` equatable and just assert it's the correct one.

Reply:
fixed in e89ecd4e32c9555e1079d9112b1168c2467ed4f4